src/sentry/incidents/logic.py

```diff
@@ -98,6 +98,12 @@
 
 CRITICAL_TRIGGER_LABEL = "critical"
 WARNING_TRIGGER_LABEL = "warning"
+INVALID_TIME_WINDOW = (
+    "Invalid time window for dynamic alert (valid windows are 15, 30, and 60 minutes)"
+)
+INVALID_ALERT_THRESHOLD = "Dynamic alerts cannot have a nonzero alert threshold"
+DYNAMIC_TIME_THRESHOLDS = {15, 30, 60}
+DYNAMIC_TIME_THRESHOLDS_SECONDS = {15 * 50, 30 * 60, 60 * 60}
```

Comment:
🤔 is this 15 * 50 on purpose instead of 15 * 60?

Reply:
and maybe `DYNAMIC_TIME_WINDOWS` instead of threshold? I keep getting the 2 mixed up so maybe you're right

Reply:
no it is not 😅